patches/components-sync-driver-data_type_manager_impl.cc.patch

```diff
@@ -0,0 +1,15 @@
+diff --git a/components/sync/driver/data_type_manager_impl.cc b/components/sync/driver/data_type_manager_impl.cc
+index f19aeafebc707eaf703b0fa71bf1c2d80dba2596..2b0a2cfe99203e331dabb6ec741649b0af4bf0aa 100644
+--- a/components/sync/driver/data_type_manager_impl.cc
++++ b/components/sync/driver/data_type_manager_impl.cc
+@@ -542,8 +542,8 @@ ModelTypeSet DataTypeManagerImpl::PrepareConfigureParams(
+   types_to_download.PutAll(clean_types);
+   types_to_download.RemoveAll(ProxyTypes());
+   types_to_download.RemoveAll(CommitOnlyTypes());
+-  if (!types_to_download.Empty())
+-    types_to_download.Put(NIGORI);
++  // if (!types_to_download.Empty())
```

Comment:
this can be reduced to a one line patch with `; // types_to_download.Put(NIGORI);`